Bottom: 556d533e255443a016709f1feafc27531fca1a6e
Top:    85920da29600dd8ddae44d66cbd3a5d8066b7658
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 14:28:33 -0500

Refresh of add-buildbot-ptxdist-worker

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index f7556ffc..3aee04d2 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -21,6 +21,19 @@
       new_user_email: buildbot@novatech-llc.com
       new_user_pass: "{{ lookup('password', '/dev/null') }}"
 
+- name: Setup Buildbot worker system
+  hosts: builder
+  become: true
+  become_user: root
+  become_method: sudo
+  tags:
+  - buildbot
+  - buildbot-worker
+  vars:
+    buildbot_master: "{{ hostvars.buildbot_master.ip_addr }}"
+  roles:
+  - role: buildbot-worker-ptxdist
+
 - name: Gitlab public keys for buildbot containers
   hosts: daedalus
   become: true
@@ -35,10 +48,9 @@
       username: buildbot
       pubkey: "{{ hostvars.buildbot_master.pubkey }}"
       title: "buildbot_master"
-
-- name: Setup Buildbot worker system
-  hosts: buildbot-worker
-  vars:
-    buildbot_master: "{{ hostvars.buildbot_master.ip_addr }}"
-  roles:
-  - role: buildbot-worker-ptxdist
+  - name: Add buildbot_worker_ptxdist pubkey to buildbot user
+    role: gitlab-sshkey
+    vars:
+      username: buildbot
+      pubkey: "{{ hostvars.buildbot_worker_ptxdist.pubkey }}"
+      title: "buildbot_worker_ptxdist"
diff --git a/roles/buildbot-worker-ptxdist/README.md b/roles/buildbot-worker-ptxdist/README.md
deleted file mode 100644
index 225dd44b..00000000
--- a/roles/buildbot-worker-ptxdist/README.md
+++ /dev/null
@@ -1,38 +0,0 @@
-Role Name
-=========
-
-A brief description of the role goes here.
-
-Requirements
-------------
-
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
-
-Role Variables
---------------
-
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
-
-Dependencies
-------------
-
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-
-Example Playbook
-----------------
-
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
-
-    - hosts: servers
-      roles:
-         - { role: username.rolename, x: 42 }
-
-License
--------
-
-BSD
-
-Author Information
-------------------
-
-An optional section for the role authors to include contact information, or a website (HTML is not allowed).
diff --git a/roles/buildbot-worker-ptxdist/handlers/main.yml b/roles/buildbot-worker-ptxdist/handlers/main.yml
deleted file mode 100644
index 78087ae9..00000000
--- a/roles/buildbot-worker-ptxdist/handlers/main.yml
+++ /dev/null
@@ -1,2 +0,0 @@
----
-# handlers file for buildbot-worker-ptxdist
\ No newline at end of file
diff --git a/roles/buildbot-worker-ptxdist/meta/main.yml b/roles/buildbot-worker-ptxdist/meta/main.yml
deleted file mode 100644
index 72237997..00000000
--- a/roles/buildbot-worker-ptxdist/meta/main.yml
+++ /dev/null
@@ -1,57 +0,0 @@
-galaxy_info:
-  author: your name
-  description: your description
-  company: your company (optional)
-
-  # If the issue tracker for your role is not on github, uncomment the
-  # next line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
-
-  # Some suggested licenses:
-  # - BSD (default)
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
-
-  min_ansible_version: 1.2
-
-  # If this a Container Enabled role, provide the minimum Ansible Container version.
-  # min_ansible_container_version:
-
-  # Optionally specify the branch Galaxy will use when accessing the GitHub
-  # repo for this role. During role install, if no tags are available,
-  # Galaxy will use this branch. During import Galaxy will access files on
-  # this branch. If Travis integration is configured, only notifications for this
-  # branch will be accepted. Otherwise, in all cases, the repo's default branch
-  # (usually master) will be used.
-  #github_branch:
-
-  #
-  # platforms is a list of platforms, and each platform has a name and a list of versions.
-  #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
-  #   - 25
-  # - name: SomePlatform
-  #   versions:
-  #   - all
-  #   - 1.0
-  #   - 7
-  #   - 99.99
-
-  galaxy_tags: []
-    # List tags for your role here, one per line. A tag is a keyword that describes
-    # and categorizes the role. Users find roles by searching for tags. Be sure to
-    # remove the '[]' above, if you add tags to this list.
-    #
-    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
-    #       Maximum 20 tags per role.
-
-dependencies: []
-  # List your role dependencies here, one per line. Be sure to remove the '[]' above,
-  # if you add dependencies to this list.
\ No newline at end of file
diff --git a/roles/buildbot-worker-ptxdist/tests/inventory b/roles/buildbot-worker-ptxdist/tests/inventory
deleted file mode 100644
index 878877b0..00000000
--- a/roles/buildbot-worker-ptxdist/tests/inventory
+++ /dev/null
@@ -1,2 +0,0 @@
-localhost
-
diff --git a/roles/buildbot-worker-ptxdist/tests/test.yml b/roles/buildbot-worker-ptxdist/tests/test.yml
deleted file mode 100644
index 2d00de44..00000000
--- a/roles/buildbot-worker-ptxdist/tests/test.yml
+++ /dev/null
@@ -1,5 +0,0 @@
----
-- hosts: localhost
-  remote_user: root
-  roles:
-    - buildbot-worker-ptxdist
\ No newline at end of file
